### PR TITLE
New Inhumen God of Greed (we did it right this time

### PR DESCRIPTION
**Pull request by Great-Beebo**
Let's hope I did it right this time

## About The Pull Request

This pull request adds the Inhumen God Qizyrot, the Mammon-Lord, God of Plots, Greed, and Wealth.

It also corrects a small grammar issue in Baotha's description

## Why It's Good For The Game

About The Pull Request

This pull request adds the Inhumen God Qizyrot, the Mammon-Lord, God of Plots, Envy, and Wealth.

It also corrects a small grammar issue in Baotha's description

Why It's Good For The Game

I would like to see mammon-maxing greedsters. I like when wealth is a center of RP, and having profit worshipping merchants haggle all day is something i like, and having their own deity to express the natural emotion of greed is only natural.

add it or dont add it.

### CODE REVIEW

**Comment by Stagrawl** [comment]
I don't know if we necessarily need more sort of... Random, loreless gods?

**Comment by pismorizzo** [comment]
Matthios is already taking up the stealing god niche.

**Comment by Coolgat3** [comment]
walmart Matthios

File: code/datums/gods/patrons/inhumen_pantheon.dm [hunk 1]
```diff
@@ -47,12 +47,24 @@
...
         "BAOTHA DEMANDS PLEASURE!",
         "LIVE, LAUGH, LOVE!",
         "BAOTHA IS MY JOY!",
 	)
+
+/datum/patron/inhumen/Qizyrot
+    name = "Qizyrot"
+    domain = "God of Plots, Greed and Wealth"
+    desc = "Man turned God, Qizyrot taught man that only through amassing great wealth can enlightenment be achieved."
+    worshippers = "Merchants, Money Lenders and Bankers
```
Comment (StrangeWeirdKitten) [inline code/datums/gods/patrons/inhumen_pantheon.dm:61]:
```suggestion
    worshippers = "Merchants, Money Lenders and Bankers."
```

**Pull request closed without merging**